src/components/ActionsPanel/components/ActionDetails.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+import { useHistory } from 'react-router-dom';
+import styled from 'styled-components';
+
+import { Button } from '@components';
+import Box from '@components/Box';
+import { Body } from '@components/NewTypography';
+import { useUserActions } from '@services';
+import { SPACING } from '@theme';
+import { ACTION_STATE, ActionTemplate } from '@types';
+
+const SBox = styled(Box)`
+  & > * {
+    margin-bottom: ${SPACING.BASE};
+  }
+`;
+
+export const ActionDetails = ({ actionTemplate }: { actionTemplate: ActionTemplate }) => {
+  const { updateUserAction, findUserAction } = useUserActions();
+  const history = useHistory();
+
+  const userAction = findUserAction(actionTemplate.name)!;
+
+  const Component = actionTemplate.Component;
+  const handleClick = () => {
+    userAction.state != 'default' &&
+      updateUserAction(userAction.uuid, { ...userAction, state: ACTION_STATE.STARTED });
+    actionTemplate.button.external
+      ? window.open(actionTemplate.button.to, '_blank', 'nopperer noreferer')
```

Comment:
Typo

Reply:
Also the links seem to be opening not in a new tab

Reply:
Í think we have a function for opening external links

Reply:
You're right ! Forgot about it’s existence